clang/lib/Driver/ToolChains/Clang.cpp

```diff
@@ -6145,6 +6149,12 @@ void Clang::ConstructJob(Compilation &C, const JobAction &JA,
       // doing the host pass.
       CmdArgs.push_back("-fsycl");
       CmdArgs.push_back("-fsycl-is-host");
+
+      // TODO add support to specify options shared between host and device
```

Comment:
As this TODO doesn't apply to this patch in particular, can this be removed from this patch/hoisted into a better more general location? @mdtoguchi

Reply:
I recall a different patch where I requested a note so we could be sure to consolidate.  I'm OK with removing from this patch.

Reply:
Mike, I filed https://github.com/intel/llvm/issues/1808 not to loose this note.

Reply:
Awesome, thanks @kbobrovs